sample-apps/auto-instrumentation-java-spark/src/main/resources/config.yaml

```
@@ -0,0 +1,9 @@
+---
+Host: "127.0.0.1"                       # Host - String Address
+port: "4567"                          # Port - String Port
```

```suggestion
Port: "4567"                          # Port - String Port
```

This should be capitalized